test_unstructured_ingest/test-ingest.sh

```diff
@@ -57,17 +57,8 @@ trap print_last_run EXIT
 
 for script in "${scripts[@]}"; do
   CURRENT_SCRIPT=$script
-  if [[ "$CURRENT_SCRIPT" == "test-ingest-notion.sh" ]]; then
-    echo "--------- RUNNING SCRIPT $script --- IGNORING FAILURES"
-    set +e
-    echo "Running ./test_unstructured_ingest/$script"
-    ./test_unstructured_ingest/"$script"
-    set -e
-    echo "--------- FINISHED SCRIPT $script ---------"
-  else
-    echo "--------- RUNNING SCRIPT $script ---------"
-    echo "Running ./test_unstructured_ingest/$script"
-    ./test_unstructured_ingest/"$script"
-    echo "--------- FINISHED SCRIPT $script ---------"
-  fi
```

Comment:
should we leave this in in some capacity so that we can quickly disable in the future?

Reply:
We should probably open up another issue to find a more dynamic approach to omit tests at run time (i.e. CLI variable)

Reply:
right, was thinking more in the meantime